modules/qt/hm.nix

```
@@ -2,11 +2,14 @@
   pkgs,
   config,
   lib,
+  osConfig ? null,
   ...
 }:
 {
   options.stylix.targets.qt = {
-    enable = config.lib.stylix.mkEnableTarget "QT" pkgs.stdenv.hostPlatform.isLinux;
+    # only autoenables on NixOS systems. see https://github.com/danth/stylix/issues/933
```

This should be more explicit:

```suggestion
    # TODO: Remove the osConfig workaround [1] ("qt: puts NixOS systemd on
    # non-NixOS distro path") once [2] ("bug: setting qt.style.name = kvantum
    # makes host systemd unusable") is resolved.
    #
    # [1]: https://github.com/danth/stylix/issues/933
    # [2]: https://github.com/nix-community/home-manager/issues/6565
```